app/models/mailboxer/notification.rb

```diff
@@ -162,8 +162,9 @@ def mark_as_deleted(participant)
 
   #Sanitizes the body and subject
   def clean
-    self.subject = sanitize(subject) if subject
-    self.body    = sanitize(body)
+    self.subject   = sanitize(subject) if subject
+    sanitized_body = String.new sanitize(body)
+    self.body      = CGI.unescapeHTML(sanitized_body)
```

Comment:
The unescapeHTML is just reverting what was done by `sanitize(body)`, right? Why bother doing anything then?